### ISSUE TITLE
nodejs component create throws warning of "not fully supported by odo" on 4.4 cluster

### ISSUE THREAD

**Issue by amitkrout** [openshift/odo#2640]
/kind bug

## What versions of software are you using?

**Operating System:**
All supported
**Output of `odo version`:**
master
## How did you run odo exactly?
```
$ odo component create nodejs
Warning: nodejs is not fully supported by odo, and it is not guaranteed to work
 ✓  Validating component [633ms]

Please use `odo push` command to create the component with source deployed
```

## Actual behavior
Throws warning

## Expected behavior
No warning message should come

**Comment by amitkrout** [comment]
more details - https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/6892/rehearse-6892-pull-ci-openshift-odo-master-v4.4-integration-e2e-benchmark/3#

**Comment by amitkrout** [comment]
ping @girishramnani, you have fixed such issues in 4.3 i guess

**Comment by kadel** [comment]
another one https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_odo/2632/pull-ci-openshift-odo-master-v4.2-integration-e2e-benchmark/681

**Comment by girishramnani** [comment]
this seems to resolved via https://github.com/openshift/odo/pull/2729

**Comment by girishramnani** [comment]
can you please confirm @amitkrout

**Comment by amitkrout** [comment]
@girishramnani Yup, it is fixed on 4.4 cluster but the pr #2729 that include this fix fails on 3.11 cluster https://github.com/openshift/odo/pull/2729#issuecomment-601120502
